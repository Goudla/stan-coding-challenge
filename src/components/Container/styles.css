.container {
  max-width: 1050px;
  margin: auto;
  padding: 0 14px;
  width: 100%;
}
