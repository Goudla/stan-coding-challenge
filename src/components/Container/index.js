import * as React from 'react';

// Styles
import './styles.css';

type Props = {
  children: React.Node
};

class Container extends React.PureComponent<Props> {
  render() {
    const { children } = this.props;

    return (
      <div className="container">
        {children}
      </div>
    );
  }
}

export default Container;
