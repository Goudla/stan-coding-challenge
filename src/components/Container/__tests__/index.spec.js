import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Scenes
import Container from '../';

describe('<Container />', () => {
  it('Should render', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<Container>Test</Container>);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
