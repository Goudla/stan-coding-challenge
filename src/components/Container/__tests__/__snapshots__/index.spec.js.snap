// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> Should render 1`] = `
<div
  className="container"
>
  Test
</div>
`;
